src/js/View/Map/Layer/MapboxGLVectorLayer.js

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 Telefónica Digital España S.L.
+// 
+// This file is part of UrboCore WWW.
+// 
+// UrboCore WWW is free software: you can redistribute it and/or
+// modify it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+// 
+// UrboCore WWW is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+// General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with UrboCore WWW. If not, see http://www.gnu.org/licenses/.
+// 
+// For those usages not covered by this license please contact with
+// iot_support at tid dot es
+'use strict';
+
+App.View.Map.Layer.MapboxGLVectorLayer = App.View.Map.Layer.MapboxGLLayer.extend({
+
+  _map: null,
+  _ids: [],
+  _idSource: '',
+  _model: null,
+  legendConfig: null,
+  dataSource: null,
+  layers: [],
+
+  initialize: function(model, body, legend, map) {
+    this._map = map;
+    this._model = model;
+    this.legendConfig = legend;
+    this._mapEvents = {};
+    this._map.addSource(this._idSource, {
+      'type': 'vector',
+      'tiles': ["https://tiles.basemaps.cartocdn.com/vectortiles/carto.streets/v1/{z}/{x}/{y}.mvt"],
```

Comment:
**Later**, we will check if we can add this layer when we have data, to avoid loading useless tiles.